Example/Core/Tests/FIRAppAssociationRegistrationUnitTests.m

```diff
@@ -52,8 +50,8 @@ - (void)testPassObject {
   id result = [FIRAppAssociationRegistration registeredObjectWithHost:host
                                                                   key:kKey
                                                         creationBlock:^id _Nullable() {
-    return obj;
-  }];
+                                                          return obj;
```

Comment:
@XiangtianDai I can't find a way. I tried ObjCBlockIndentWidth from https://clang.llvm.org/docs/ClangFormatStyleOptions.html, but it only adjusts the indentation from the line above from 2 to the value that is set. 

It seems ugly, but I'm not sure we should diverge from the Google style.